indra/llui/llscrolllistctrl.cpp

```diff
@@ -423,6 +423,19 @@ std::vector<LLScrollListItem*> LLScrollListCtrl::getAllSelected() const
     return ret;
 }
 
+std::vector<LLSD> LLScrollListCtrl::getAllSelectedValues() const
```

Comment:
I like introducing `getAllSelectedValues()`; I'm a little surprised that didn't already exist. But I do wonder why you return a `std::vector<LLSD>` instead of an LLSD array?